LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
How come not mit?

Reply:
That's a [restriction from Google](https://developers.google.com/tag-manager/templates/gallery#prepare_your_project_files).